src/library/recording/dlgrecording.cpp

```diff
@@ -160,20 +163,21 @@ void DlgRecording::slotRecordingEnabled(bool isRecording) {
 void DlgRecording::slotBytesRecorded(int bytes) {
     double megabytes = bytes / 1048576.0;
     m_bytesRecordedStr = QString::number(megabytes,'f',2);
-    refreshLabel();
+    refreshLabels();
 }
 
 // gets recorded duration and update label
 void DlgRecording::slotDurationRecorded(QString durationRecorded) {
     m_durationRecordedStr = durationRecorded;
-    refreshLabel();
+    refreshLabels();
 }
 
 // update label besides start/stop button
-void DlgRecording::refreshLabel() {
-    QString text = tr("Recording to file: %1 (%2 MiB written in %3)")
-              .arg(m_pRecordingManager->getRecordingFile())
-              .arg(m_bytesRecordedStr)
-              .arg(m_durationRecordedStr);
-    label->setText(text);
- }
+void DlgRecording::refreshLabels() {
+    QString recFile = m_pRecordingManager->getRecordingFile();
+    QString recData = tr("(%1 MiB written in %2)")
```

Comment:
Maybe we should move the parentheses out of the translation string to not bother our translators. Also, if we ever redesign and wan't to get rid of the parentheses, we don't have to retranslate.

```suggestion
    QString recData = (QStringLiteral("(") + tr("%1 MiB written in %2") + QStringLiteral(")"))
```

Reply:
Whoops sorry, I forgot to copy the QString before the brackets.